.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,32 @@
+name: Feature Request
```

Comment:
Do we want to use the term "Feature Request" or is there something better?

Reply:
Enhancement has a meaning that's not relevant here.  In JIRA we use it for improvements that bring direct user benefit but are not user controllable. I'd prefer not to confuse that.

'Feature' seems reasonable

Reply:
BTW 'Feature Request' seems ok to me too if we want to follow KISS principles and align with JIRA. But I get how the 'Request' part is odd here as these issues are never requests that someone do something, they are statements of intent to do something.